### ISSUE TITLE
[BUG] Kodi crashing as soon as try to play content 

### ISSUE THREAD

**Issue by 666jones** [jurialmunkey/skin.arctic.fuse#65]
### Skin section

Other

### Current Behavior

As soon as I try to play content Kodi crashes and quits out. It also happens when I try to open add-on like log viewer for example. I installed the updates one by one and it only started happening after the fuse update. So it's definitely nothing Todo with the tmdb helper update or skin variables update.

I have included a log file

### Expected Behavior

Not to crash :)

### Steps To Reproduce

1. Install update play a movie or TV show.

### Screenshots and Additional Info

[kodi.log](https://github.com/jurialmunkey/skin.arctic.fuse/files/12239810/kodi.log)

### Checklist

- [X] I have made a separate issue for each bug that I am reporting.
- [X] I have searched the issues and this bug has not been reported.
- [X] I am using the most recent skin version.
- [X] I have confirmed that this bug does not occur in the default skin.

**Comment by jurialmunkey** [comment]
Do you have the crashlog with a stack trace?

**Comment by 666jones** [comment]
> Do you have the crashlog with a stack trace?

Where can I find the crash log? I just done a quick Google to find the answer but it seems to be pointing me to the kodi.log file

**Comment by 666jones** [comment]
> I'm not sure on Android as I don't use it but standard log is not much use for a segfault where Kodi exits suddenly. Kodi has to be running so that it can write out log messages -- so all the standard log can have is messages about events which did _not_ cause the crash.
> 
> Need a crash log with a stack trace to know which process was being operated on at the time of the crash.

Ok I'm still looking on Google to find out about how to obtain this crash log now. I have checked all my Kodi folders and cannot find anything referencing (crash) or similar. I found kodi.old.log but that's it. I'll keep digging and update you as soon as I have the correct log

**Comment by jurialmunkey** [comment]
The other thing you can do is go back through the commit history and checkout individual commits to see where the crash started exactly. If you can pinpoint which commit it is then that will give me a good idea of the cause.

You can see the commit history between the two versions here:
https://github.com/jurialmunkey/skin.arctic.fuse/compare/v0.1.2...v0.1.5

If you click this button on the commit it will take you to a version at that exact point
![image](https://github.com/jurialmunkey/skin.arctic.fuse/assets/4618719/48962642-d590-4ea6-8294-f475635b7fe4)

Then you can use a the green "code" button to download a zip of that exact commit to install
![image](https://github.com/jurialmunkey/skin.arctic.fuse/assets/4618719/61d2860e-07f6-47a8-a28e-6d9e0ced9db7)

You can see in the top left that you're on the commit you looked at and then download a zip of the repo at that point. If you install the zip then you'll be able to tell.

Basically pick a commit in the middle somewhere and install it and see, then use a hotter/colder approach till you find where abouts it happened.

**Comment by jurialmunkey** [comment]
The ones I'd try first is the commit immediately before the ratings change
https://github.com/jurialmunkey/skin.arctic.fuse/tree/42e4ffbbec12b1cb022d6180b267f56487255804

And the one before the fallback info
https://github.com/jurialmunkey/skin.arctic.fuse/tree/599c3b960cb0dab38cde5b7768eee40d82fe755d

**Comment by 666jones** [comment]
> The ones I'd try first is the commit immediately before the ratings change https://github.com/jurialmunkey/skin.arctic.fuse/tree/42e4ffbbec12b1cb022d6180b267f56487255804
> 
> And the one before the fallback info https://github.com/jurialmunkey/skin.arctic.fuse/tree/599c3b960cb0dab38cde5b7768eee40d82fe755d

Ok just testing the first one now. Then I'll treat the ones you mentioned above. Thanks for your help much appreciated

**Comment by 666jones** [comment]
> The ones I'd try first is the commit immediately before the ratings change https://github.com/jurialmunkey/skin.arctic.fuse/tree/42e4ffbbec12b1cb022d6180b267f56487255804
> 
> And the one before the fallback info https://github.com/jurialmunkey/skin.arctic.fuse/tree/599c3b960cb0dab38cde5b7768eee40d82fe755d

https://github.com/jurialmunkey/skin.arctic.fuse/tree/599c3b960cb0dab38cde5b7768eee40d82fe755d

This one has the crash issue


https://github.com/jurialmunkey/skin.arctic.fuse/tree/42e4ffbbec12b1cb022d6180b267f56487255804

This one works.

Do you want me to narrow it down some more?

**Comment by 666jones** [comment]
I just looked and there is alot more version. I will narrow it down more

**Comment by 666jones** [comment]
> The ones I'd try first is the commit immediately before the ratings change https://github.com/jurialmunkey/skin.arctic.fuse/tree/42e4ffbbec12b1cb022d6180b267f56487255804
> 
> And the one before the fallback info https://github.com/jurialmunkey/skin.arctic.fuse/tree/599c3b960cb0dab38cde5b7768eee40d82fe755d

f3f7b29 = working
41e7dad = crashes 

Hope this helps

**Comment by 666jones** [comment]
> @666jones - Can you try that? https://github.com/jurialmunkey/skin.arctic.fuse/tree/48c6de9700e9973d903f9c92bb76c8598b869b26
> 
> I think it was ending up in an endless loop of getting the label that it was also setting

I can confirm that 48c6de9 has fixed the crashing issue. Very lovely

**Comment by jurialmunkey** [comment]
Excellent! Thanks so much for testing to narrow that down - was really helpful. I'll push that to the repo now.
